headers/augment_headers.py

```diff
@@ -289,6 +293,13 @@ def add_header_content(
         action="store_true",
         help="add alias declarations",
     )
+    parser.add_argument(
+        "-ad",
+        "--mark_aliases_deprecated",
+        action="store_true",
+        help="mark aliases as deprecated (if --aliases is set)",
```

Comment:
Let's mention the syntax specifically:

```suggestion
        help="mark aliases with __attribute__((deprecated)) if --aliases is set",
```